client/src/components/course-view/ModelsView.tsx

```diff
@@ -252,19 +263,40 @@ const ModelsView = (): JSX.Element => {
                 secondaryAction={
                   editRights ? (
                     <>
-                      <IconButton
-                        onClick={() =>
-                          handleArchiveModel(model.id, !model.archived)
+                      <Tooltip placement="top" title="Edit assessment model">
```

Comment:
It becomes fairly obvious once you open the dialog, but should this specify that it's for editing the name specifically? The actual editing is done through the graph, right?

Reply:
I agree, "Edit name" or "Rename" would be more clear. If we add more stuff later we can call it edit info.